### DISCUSSION TITLE
https://ericswpark.com/blog/2022/2022-12-14-ffmpeg-convert-hdr-to-sdr/

### DISCUSSION THREAD

**Discussion by giscus** [ericswpark/ericswpark.github.io-comments#13] [Announcements]
# https://ericswpark.com/blog/2022/2022-12-14-ffmpeg-convert-hdr-to-sdr/

Student Developer

https://ericswpark.com/blog/2022/2022-12-14-ffmpeg-convert-hdr-to-sdr/

**Comment by gihayes** [comment]
Great Post! I've been wrestling over which one of these to use. You confirmed my observations. Hable too dark. And Mobius too vivid. I understand BT.2390 is not used much anymore, and the software I have only performs software tone mapping with it. I am gonna search for some samples of it to see if I should stick with Reinhard or BT for software tone mapping. Have you ever used BT.2390?

**Reply by ericswpark** [reply]
I haven't used BT.2390, but IIRC Jellyfin now uses it over the other tonemapping algorithms mentioned in the post.

**Reply by gihayes** [reply]
Thanks for the reply. I'm using Emby of which Jellyfin is a branch.

**Comment by notsmoothsteve** [comment]
Thanks for this article!  It's been helpful to me.
Definitely no one-size-fits-all, and my use case (converting HDR footage for use in SDR videos) may be on the unusual side.  One thing I've found in playing with settings is that the NPL makes a huge difference, not just to the brightness (obvs) but also to the colour mapping.  So for anyone else who stumbles upon this blog like I did when figuring this stuff out, test different NPLs as well as different algorithms and different desat settings.
